packages/react-components/src/localization/locales.ts

```diff
@@ -1,31 +1,49 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import en_US from './translated/en-US.json';
-import de from './translated/de.json';
-import ar from './translated/ar.json';
 import { Locale } from './LocalizationProvider';
+import _en_US from './translated/en_US.json';
+import _en_GB from './translated/en_US.json'; // TODO: This is temporary

```

Comment:
Errr.. why?

Better to not support it. We might forget to clean this up before some release... then ouch :/